paper/src/main/java/rocks/gravili/notquests/paper/conversation/ConversationManager.java

```diff
@@ -39,6 +35,7 @@
 import rocks.gravili.notquests.paper.conversation.interactionhandlers.SendClickableText;
 import rocks.gravili.notquests.paper.conversation.interactionhandlers.ConversationInteractionHandler;
 import rocks.gravili.notquests.paper.managers.data.Category;
+import rocks.gravili.notquests.paper.managers.npc.CitizensNPC;
```

Comment:
Don't import CitizensNPC here. Everything outside of integration-specific classes needs to be as generic as possible so we can support more npc plugins easier. Instead, maybe add a method to NqNPC which runs during playConversation and implement that inside of CitizensNPC. That way all Citizens-specific logic can stay in one class